.github/workflows/ci-app.yml

```
@@ -56,3 +56,6 @@ jobs:
           sourceDir: app
           coverageFile: app/coverage_report.xml
           token: ${{ secrets.GITHUB_TOKEN }}
+          thresholdAll: 0.8
+          thresholdNew: 0.9
+          thresholdModified: 0.8
```

```suggestion
          thresholdModified: 0.8
```

Is this file specific -- in other words, does this mean edits to a file with no pre-existing coverage will fail this check if the PR doesn't take coverage from 0 -> 80%?

---

See https://github.com/navapbc/labs-decision-support-tool/pull/83#issuecomment-2386124218

---

I added an exclusion for app.py:
https://github.com/navapbc/labs-decision-support-tool/pull/83/commits/2f13d8e62102632e5fb8572f4d6ec9aaf2fc75ea

---

So, I'd recommend that files with no pre-existing coverage be added to the `omit` list rather than changing the threshold.